src/components/dataSet-settings.js

```diff
@@ -59,11 +59,15 @@ class DataSetSettings extends React.Component {
             this.dataSetToChange = args[0].name
             const argsData = args[0]
             this.setState({
-                specificPeriodDSDownload: argsData.specificPeriodDSDownload,
-                specificPeriodDSDBTrimming: argsData.specificPeriodDSDBTrimming,
-                specificSettingName: argsData.id,
+                specificSetting: {
+                    specificSettingName: argsData.id,
```

Comment:
It seems a bit strange to call this `specifiicSettingName` when it is under the `specificSetting` object, should this just be `name`?